hotnews/index.html

```diff
@@ -1,37 +1,107 @@
+        fontFamily: {
+          sans: ["Inter", "sans-serif"]
+        },
+        minheight: '90px',
+        extend: {
+          padding: {
+            paddingleft: '22px'
+          },
+          width: '1349px',
+          height: {
+            height: '90px',
+            height2: '52px',
+            height3: '22px',
+            height4: '50px'
+          },
+          colors: {
+            hotblue: '#2145ac',
+            hotgreen: '#3f9863',
+            hotred: '#eb3223',
+            hotblack: '#393939',
+            hotEconomie: '#ed5c43',
+            hotSport: '#36a562',
+            hotLife: '#d0459e',
+            hotScience: '#767fea',
+            hotRevista: '#484848',
+            hotDosare: '9d8562',
+            hotHome: '2145ac',
+            hotWhite: '#fff',
+            hotWar: '#ffa900'
           }
         }
       }
-    </script>
-  </head>
-  <body>
-    <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium">
-      <div class="flex grow items-center">
-        <img src="https://hotnews.ro/static/assets/icons/ic_updated.svg" alt="updated_icon">
-        <div class="text-hotblue">Actualizat acum un minut</div>
-      </div>
-      <div class="flex shrink">
-        <div class="text-hotblue pr-1">EUR</div>
-        <div class="text-hotgreen pr-1">4.9395</div>
-        <div class="text-hotblue pr-1">USD</div>
-        <div class="text-hotred">4.8195</div>
-      </div>
+    }
+  </script>
+</head>
+
+<body>
+  <div class="flex max-w-5xl mx-auto px-4 py-1.5 text-sm font-medium text-[14px]">
+    <div class="flex grow items-center">
+      <img src="https://hotnews.ro/static/assets/icons/ic_updated.svg" alt="updated_icon">
+      <div class="text-hotblue">Actualizat acum un minut</div>
     </div>
-    <div class='bg-hotblue h-10'>
+    <div class="flex shrink text-[14px]">
+      <div class="text-hotblue pr-1">EUR</div>
+      <div class="text-hotred pr-1">4.9395</div>
+      <div class="text-hotblue pr-1">USD</div>
+      <div class="text-hotgreen">4.8195</div>
     </div>
-  </body>
+  </div>
+  <div class="bg-hotblue">
+    <div class='mx-auto max-w-5xl h-[90px] flex'>
+      <div class="items-center justify-between flex grow">
+        <a href="../hotnews/index.html">
+          <img class="h-[52px] cursor-pointer" src="https://www.hotnews.ro/static/assets/logos/hotnews_header.svg"
+            alt="hotnews-logo"></a>
+        <div class="flex shrink">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_search.svg" alt="Search-Icon">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_user.svg" alt="User-Icon">
+          <img class="m-2.5 h-[22px]" src="https://www.hotnews.ro/static/assets/icons/ic_menu.svg" alt="Burger-Menu">
+        </div>
+      </div>
+    </div>
+  </div>
+  <section
+    class="p-[22px] text-hotblack mx-auto text-[18px] font-bold font-['Inter', 'sans-serif'] flex  h-14 items-center max-w-5xl">
```

Comment:
**font-bold font-['Inter', 'sans-serif']** can be replaced with your defined rule **font-sans**

Reply:
I removed font-['Inter', 'sans-serif'], there was no need to add font sans on classes because whole document uses Inter, sans-serif. In tailwind config at fontFamily is already there.